devenv/frontend-service/build-grafana.sh

```diff
@@ -6,7 +6,7 @@ echo "Go build cache: $(go env GOCACHE), $(ls -1 $(go env GOCACHE) | wc -l) item
 # Need to build version into the binary so plugin compatibility works correctly
 VERSION=$(jq -r .version package.json)
 
-go build \
+go build -v \
```

Comment:
do we definitely want this, or was it just for debugging?
i have no opinion either way, just checking 😄

Reply:
either way. it was for debugging/reassurance of what build was doing, that i figured we can just leave in.